TypeInjections/Tool/README.md

```diff
@@ -1,21 +1,23 @@
 ## typeCart features
 typeCart CLI offers two ways to compare projects:
 - projects on local computer:
-  `$ typecart local [--old <path>] [--new <path>] [--print <path>] [--file <names>]`
+  `$ typecart local [--old <path>] [--new <path>] [--print <path>] [--ignore <names>]`
     * Required Options
       * `-o` `--old` Give the absolute path of old project
       * `-n` `--new` Give the absolute path of new project
       * `-p` `--print` Tell typeCart which folder to print out generated files
   * Additional Options
+    * `-i` `--ignore` Give list of filenames to ignore when running typeCart
     * (coming soon) `-e` `--entrypoint` Specify certain file(s) to run typeCart on 
-- git commits in repository: `$ typecart git [--old <commit>]  [--new <commit>] [--print <path>] [--clone <git URL>] [--file <names>]`
+- git commits in repository: `$ typecart git [--old <commit>]  [--new <commit>] [--print <path>] [--clone <git URL>] [--ignore <names>]`
   * Required options 
     * `-o` `--old` Give the absolute path of old project
     * `-n` `--new` Give the absolute path of new project
     * `-p` `--print` Tell typeCart which folder to print out generated files
   * Additional Options
     * `-c` `--clone` Provide git URL to checkout commits 
-    * `-e` `--entrypoint` Specify certain file(s) to run typeCart on
+    * `-i` `--ignore` Give list of filenames to ignore when running typeCart 
+    * (coming soon) `-e` `--entrypoint` Specify certain file(s) to run typeCart on
```

Comment:
Once we have `entrypoint` do we still need `ignore`, and vice versa? In what cases do we need to ignore things that files in `entrypoint` depends on?

Reply:
This is an important point to clarify: is `--entrypoint` a file or a directory?

If it is a directory, then the ability to ignore certain files is important (and we have already implemented that in the API).

Reply:
Yeah this is a good question. The `--entrypoint` feature was a nice idea suggested by Rachel. The idea is `typecart git` will run typeCart on the entire repository and the output files will create way too many casting functions.

My original plan for `--entrypoint` was to allow the user to input a file or directory (if input had a file extension, then it's a file) then run typeCart only on that file/directory. 

The `--ignore` function can be used with `--entrypoint` if the user inputs a folder. If the user is trying to run a specific file, `--ignore` would be useless.


Does this approach seem reasonable? I would appreciate any feedback! @txiang61 @ruijiefang

Reply:
The typecart API (implemented as the `Typecart` class in `Typecart.fs`) is designed in a way such that the user does not have to perform any I/O operations manually. The API will auto-resolve the path to directories or files. As such, I think just having two paths as inputs will suffice. I don't see the need for a special argument.

Reply:
Yeah I can remove the option for `typecart local`.

But for `typecart git`, `--entrypoint` is the only way to specify a file or folder. I will update readme accordingly